service/models.py

```diff
@@ -140,7 +140,7 @@ def deserialize(self, data: dict):
         try:
             self.product_id = data["product_id"]
             self.condition = getattr(Condition, data["condition"])
-            self.can_update = UpdateStatusType.ENABLED
+            self.can_update = getattr(UpdateStatusType, data["can_update"])
```

Comment:
Why are we deserializing can_update? The reason I changed it to UpdateStatusType.ENABLED is because we have an action to set the update status. Otherwise the default should just be ENABLED right?

Reply:
I made this change because when I create a new item in the GUI and set the can_update drop down menu to DISABLED, I noticed the list had ENABLED in it. It was b/c of this line, it kept setting can_update to ENABLED, so it was pretty much ignoring what I requested in the UI

The existing TDD unit tests pass, so I think this change is ok?

Reply:
Got it. I guess I was under the assumption that we don't want the user to be able to change this attribute through the regular CRUD API and only change it through the actions.
I think it's fine to leave it we all agree that we want to treat it like a regular attribute.